tests/phpunit/tests/post/nav-menu.php

```diff
@@ -383,12 +384,14 @@ public function test_wp_get_nav_menu_items_cache_primes_terms() {
 
 		$start_num_queries = get_num_queries();
 		wp_get_nav_menu_items( $this->menu_id );
+		get_term_meta( $term_ids[0] );
 		$queries_made = get_num_queries() - $start_num_queries;
 		$this->assertSame( 6, $queries_made, 'Only does 6 database queries when running wp_get_nav_menu_items.' );
 
-		$args = $action_terms->get_args();
-		$last = end( $args );
-		$this->assertSameSets( $term_ids, $last[1], '_prime_term_caches() was not executed.' );
+		$args       = $action_terms->get_args();
+		$first      = reset( $args );
+		$term_ids[] = $this->menu_id;
+		$this->assertSameSets( $term_ids, $first[1], '_prime_term_caches() was not executed.' );
```

Comment:
What's the purpose of this change?

Reply:
@peterwilsoncc Now that all term meta is lazy loaded and after [this](https://github.com/WordPress/wordpress-develop/commit/7130af403fff492d4bde3e2fbd08bc3d25150dfb) change, the first element in this array, primes the menu's term meta and all other term ids term meta in one go.